test/cmocka/src/audio/buffer/mock.c

```diff
@@ -57,6 +57,13 @@ void *rballoc(int zone, uint32_t caps, size_t bytes)
 	return malloc(bytes);
 }
 
+int memcpy_s(void *dest, size_t dest_size,
+	     const void *src, size_t src_size)
+{
+	memcpy(dest, src, src_size);
```

Comment:
If it's named with `_s` there should be some checks, even if it's just in unit tests. Feel free to copy it from host implementation. You can move it to some lib that is visible in all unit tests and use it there instead of putting it in mock for every compilation unit that needs memcpy_s